Tasks/ServiceFabricUpdateManifestsV2/Update-DockerImageSettings.psm1

```diff
@@ -61,7 +62,8 @@ function Update-DockerImageSettings
             }
             else
             {
-                $imageName = $imageDigestRepoName
+                $imageName = $imageDigestName
+                $imageNameToDigestMapping[$imageDigestRepoName] = $imageDigestValue
```

Comment:
it would be better to add a condition to check whether imageDigestName and imageDigestRepoName are not same and only then add this entry to hash

Reply:
In the dictionary it will automatically overwrite if keys will be same.